.env.example

```diff
@@ -0,0 +1,9 @@
+TOKEN=YOUR_BOT_TOKEN_HERE
```

Comment:
This file will hold your config, this is to make sure you don't push any configuration to your image and dockerhub.

The image created below is a stateless image without configuration